3574/1/table.rb

```diff
@@ -0,0 +1,18 @@
+require 'terminal-table'
+
+class Output
+  def call(data)
+    rows = create_rows(data)
+    create_table(data, rows)
+  end
+
+  private
+
+  def create_rows(data)
+    data.each_with_object([]) { |gem, stock| stock << gem.values_at(*HEADER) }
+  end
+
+  def create_table(data, rows)
+    puts Terminal::Table.new headings: HEADER, rows: rows unless data.empty?
```

Comment:
I think create_table should return just table. And you can do `puts table` in your `call` method